tests/app/forms/test_questionnaire_form.py

```diff
@@ -178,7 +178,7 @@ def test_answer_errors_are_mapped(self):
 
             form.validate()
             answer_errors = form.answer_errors('total-retail-turnover-answer')
-            self.assertIn(error_messages["NUMBER_TOO_SMALL"] % dict(min=0), answer_errors)
+            self.assertIn(error_messages["NUMBER_TOO_SMALL"] % dict(min='£0.00'), answer_errors)
```

Comment:
I think it would be clearer if we changed the answer to one that generates an error message that doesn't require interpolation of a value e.g. using `a` instead of `-1` would generate a `INVALID_NUMBER` error.

Reply:
@ajmaddaford I think as the test is to map a previous answer to an error, i think using a default error message goes against the test itself. Maybe we should add a new test for this case?

Reply:
My bad, I mis-understood the test. 

Having said that I think it would be clearer if we changed the answer to be a number e.g. `total-number-employees` rather than a currency answer.

I also think we should rename the test, as it's not mapping any errors (some previous tests are testing the `map_errors` method). Something like `test_answer_errors_are_interpolated` would be clearer.